cypress/integration/sucuri-scanner.js

```diff
@@ -205,12 +205,12 @@ describe( 'Run integration tests', () => {
 		cy.get('.sucuriscan-alert').contains('Automatic Secret Keys Updater disabled.');
 	});
 
-	it('can whitelist blocked PHP files', () => {
+	it('can unblocked PHP files', () => {
```

Comment:
I am not the best english speaker to suggest that, but I think the word "unblocked" here could be just "unblock".